envelope.json

```diff
@@ -0,0 +1,70 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://mo-rise.github.com/brefv/envelope.json",
+    "type": "object",
+    "title": "Envelope",
+    "description": "An envelope including relevant timestamps, sender id and data type for any kind of message",
+    "examples": [
+      {
+        "sampled_at": 1632746168.503,
```

Comment:
Hmm, the intention with `sampled_at` was for it to contain the timestamp for the time which the content in `message` was sampled, i.e. when we got the data from the sensor, the idea was to then not have any timestamp within the message. But this might be a bad idea, I dont really know. 

`sent_at` might be better renamed as `sealed_at` according to your suggestion? Since an envelope can be sent and resent several times without opening it it might be tricky anyway to define a single timestamp for when it was sent?

Yes, the different times, for example with simulations, is something that needs to be given some thoughts. I have no answers at this point, only further questions:
* should we allow simulations in both realtime and faster-than-realtime? How to handle timings with simulations with faster-than-realtime execution?
* How should we handle envelopes from sources with different delay? Is it merely a frontend thing? i.e. the delays hould be highlighted to the user?

Reply:
This very much connects to how we want to handle:
* recording
* replay of recorded data
* syncing of data streams (should we?)
* visualization of data (with/without delays?)